### ISSUE TITLE
Can't set props that aren't strings

### ISSUE THREAD

**Issue by ggarnier** [scup/atellier#22]
When I try to live edit a prop that isn't a string (like a function or an object), I get this warning:

```
Warning: Failed propType: Invalid prop `something` of type `string` supplied to `MyComponent`, expected `object`.
```

Seems like prop values are being automatically converted to strings.

**Comment by agutoli** [comment]
@ggarnier, we just released a version v0.1.9, can you try now? Now you do not need to pass React to Atellier, ex.

`import Atellier from 'react-atellier';`

and 
`<Atellier components={[]} />`

tks!

**Comment by ggarnier** [comment]
@agutoli now it's working. Thanks!
